src/lib/responses.ts

```diff
@@ -116,4 +121,24 @@ const getUserResponse = async (
   return values[arr.findIndex(val => !!val)];
 };
 
-export { submitResponse, removeAll, getAll, getUserResponse };
+const getResponsesCount = async (
```

Comment:
Change this function to take an array of pids and return an array of `ResponsesCount`s. You'll have to flat map each pid into the array of keys where each triplet corresponds to one pid. Then you'll have to process the resulting array, grouping each triplet into a `ResponsesCount` object.

Reply:
Created an overload to accept both, array and number. This way, the method call is easier to use in the event API, where both are used (Once you need an array and every other call is a number as argument).

Reply:
Well done!